bluemira/base/components.py

```diff
@@ -25,16 +25,14 @@
 
 from __future__ import annotations
 
+from typing import Any, List, Optional, Union
+
 import anytree
 from anytree import NodeMixin, RenderTree
-from typing import Any, List, Optional, Union
 
-from bluemira.display.plotter import Plottable
+from bluemira.base.error import ComponentError
 from bluemira.display.displayer import DisplayableCAD
-
-import bluemira.geometry as geo
-
-from .error import ComponentError
+from bluemira.display.plotter import Plottable
 
 
```

Comment:
Can we include an `if TYPE_CHECKING:` block so that IDEs can still pick up what the actual class being used is? We'll also need to import `TYPE_CHECKING` from `typing`.

```suggestion
if TYPE_CHECKING:
    from bluemira.geometry.base import BluemiraGeo

```

The `from __future__ import annotations` will treat the annotations as strings so that things will run, but the neat benefits of IDEs being able to work out what a class can do would be lost if we remove the import altogether.